corehq/apps/app_manager/tests/data/suite/suite-case-detail-instances.xml

```diff
@@ -62,7 +62,7 @@
     <field>
       <header>
         <text>
-          <locale id="m0.case_short.case_name_2.header"/>
+          <locale id="m0.case_short.case_calculated_property_2.header"/>
```

Comment:
Is using the case name in case detail now interpreted as an xpath calculation? How did that happen? Doesn't seem like a problem - as long as the "Calculated Property" label doesn't appear - just curious.

Reply:
It comes from this calculate column:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/tests/data/suite/app_case_detail_instances.json#L589

Reply:
Oh, it's not all case names then, cool.